coalib/output/Logging.py

```diff
@@ -1,16 +1,15 @@
 from datetime import datetime
-import json
 import io
+import json
 import logging
 import logging.config
+import sys
```

Comment:
https://github.com/coala/coala/pull/3622#discussion_r97488525
Should I change it back?